src/registrar/templates/includes/portfolio_status_manage.html

```diff
@@ -0,0 +1,7 @@
+{% extends "includes/request_status_manage.html" %}
```

Comment:
Why do we need an interstitial that only passes the baton to another include? Can't this block live in domain_request_status.html directly?